content/docs/user-guide/basic-concepts/data-artifact.md

```diff
@@ -6,3 +6,5 @@ match: ['data artifact', 'data artifacts', 'artifact', 'artifacts']
 Any data file or directory, as well as intermediate or final result that is
 tracked by DVC, for example by using `dvc add` or `dvc run`. See
 [Versioning Data and Models](/doc/use-cases/versioning-data-and-model-files).
+
+A.k.a. _output_
```

Comment:
tbh, not sure when data artifact became a synonym  ... it's not exactly true ... e.g. we can depend on some data artifact that is not tracked by DVC whatsoever

Reply:
Good point. I've been meaning to address these redundant terms for a while in fact (see https://github.com/iterative/dvc.org/issues/608). OK, I remove the cross-ref for now.